adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +319,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
+        return mIsExtendedLifeTimeToken;
+    }
+
+    /**
+     * Sets the flag to indicate whether the token being returned is a token only
+     * valid in terms of extended lifetime
+     *
+     * @param isExtendedLifeTimeToken
+     */
+    final void setIsExtendedLifeTimeToken(final boolean isExtendedLifeTimeToken) {
+        mIsExtendedLifeTimeToken = isExtendedLifeTimeToken;
+    }
+
+    final void setExtendedExpiresOn(final Date extendedExpiresOn) {
+        mExtendedExpiresOn = extendedExpiresOn;
+    }
+
+    final public Date getExtendedExpiresOn() {
```

Comment:
everywhere in this file

---
In reply to: [70890162](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70890162) [](ancestors = 70890162)